app/controllers/diaper_drive_participants_controller.rb

```diff
@@ -6,16 +6,28 @@ def index
   def create
```

Comment:
Can you refactor this to use `respond_to` blocking?

see: http://guides.rubyonrails.org/working_with_javascript_in_rails.html#an-introduction-to-ajax#a-simple-example

The `format.js` block is the one that will handle the ajax request -- all XHR stuff can go in there.